.eslintrc

```diff
@@ -18,5 +19,8 @@
     "prefer-rest-params": 0,
     "prefer-reflect": 0,
     "valid-jsdoc": 0
+  },
+  "globals": {
+    "global": true
```

Comment:
@lemonmade `global` is used to shim in certain functionality if needed in Node. https://github.com/Shopify/js-buy-sdk/blob/master/src/metal/global.js